bazel/init/stage_1.bzl

```diff
@@ -70,8 +78,8 @@ def stage_1():
     maybe(
         name = "com_google_absl",
         repo_rule = http_archive,
-        strip_prefix = "abseil-cpp-20240722.1",
-        urls = ["https://github.com/abseil/abseil-cpp/archive/refs/tags/20240722.1.tar.gz"],
+        strip_prefix = "abseil-cpp-20250127.0",
```

Comment:
A previous change to this repo (in #1162) removed `bazel/dependencies/abseil:0001-remove-maes-and-msse4.1-option-from-cross-compilation.patch`.  Do we know that the ARM build issue is solved in current Abseil?  And if we are removing the patch, should we also delete the diff from the repo?

Reply:
Answering my own question: yes, this is fixed in current absl, as of https://github.com/abseil/abseil-cpp/commit/4e0956109640efa4b2ee68a2d6640fb13afa420e

But could you still remove https://github.com/enfabrica/enkit/tree/master/bazel/dependencies/abseil ?

Reply:
Done. I removed the unused patch. I did not experience the `-maes` compilation flag issue when I built `bazel build --override_repository=enkit=$HOME/enkit //sdk //systest //ib` with the latest unpatched abseil package.